docs/containers-containerfile.5.md

```diff
@@ -0,0 +1,505 @@
+% "CONTAINERFILE" "5" "MAY 2014" "" "Container User Manuals"
```

Comment:
Is this date right, and/or do we need to credit Docker?

Reply:
Updated.